utils/_context/containers.py

```diff
@@ -248,8 +282,10 @@ def __init__(self, host_log_folder, proxy_state) -> None:
             },
             working_dir="/app",
             volumes={
-                f"./{host_log_folder}/interfaces/": {"bind": f"/app/{host_log_folder}/interfaces", "mode": "rw",},
-                "./utils/": {"bind": "/app/utils/", "mode": "ro"},
+                f"{self.host_project_dir}/{host_log_folder}/interfaces/": {
```

Comment:
You can keep the `.` here, and handles `host_project_dir` in `_fix_host_pwd_in_volumes` : 

```
host_pwd = os.getcwd()

# replaced by

host_pwd = os.environ.get("HOST_PROJECT_DIR", os.getcwd())
```

It will keep the volume definition simpler, sticking to the docker compose logic.